mobopt/_bayes.py

```diff
@@ -622,4 +715,49 @@ def ReadSpace(self, filename="space.npz"):
 
         self.vprint("Read data from "+filename)
 
-        return
\ No newline at end of file
+        return
+
+    def penalty(self, front, y_pot, eps, ref_point):
+
+        """
+        Penalty calculation
```

Comment:
La funzione non restituisce sempre una penalty, quindi il nome è impreciso, quello di prima (`__hv_contrib`) andava benissimo. Quando dicevo di rinominare in `penalty`, parlavo della variabile `p` più sotto, che avete sistemato e va bene così. (Notate che un commento si riferisce alla singola riga di codice che sta sopra il commento stesso, a meno che non sia indicato diversamente, come in questo commento qua)